systems/__main__.py

```diff
@@ -1,28 +1,37 @@
 from threading import Thread
 
 from systems.clients import CanbusNet, PiNet
-from systems.core import ControllerWorker, CoreServer
+from systems.core import BrowserProxy, ControllerWorker, CoreServer
 
 # In-proc transports for multi-threaded application, use ipc for multi-processed.
 # NOTE: in `inproc` transport, ensure server is started before anything
 
 _frontend_address = "inproc://frontend"
 _backend_address = "inproc://backend"
+_browser_address = "ws://127.0.0.1:8001"
 
 
 def start_systems():
     """Set up all threads with respective tasks."""
     core_server = CoreServer(_backend_address, _frontend_address)
     controller_worker = ControllerWorker(_backend_address)
-    pi_net = PiNet(_frontend_address)
+    browser_proxy = BrowserProxy(_frontend_address, _browser_address)
+    pi_net = PiNet(_browser_address)
```

Comment:
Do we need to run `PiNet` if it's deprecated? We could just leave it out for now and only run it if needed

Reply:
i think it might be helpful to leave it in fot rn while we're integrating just because it essentially acts as a mock for how the UI will communicate, feel free to lmk tho i'm down to take it out